lib/API/JSON/v1/CustomFields.php

```diff
@@ -45,7 +45,9 @@ function save($data = []) {
     }
     $custom_field = CustomField::findOne($custom_field->id);
```

Comment:
Maybe we could use the newly created response builder here. We are already re-fetching the model from DB...